easylist_adult/adult_specific_hide.txt

```diff
@@ -1256,6 +1256,7 @@ tokyomotion.com##.is-gapless > .has-text-centered:has(> div > .adv)
 vagina.nl##.list-item:has([data-idzone])
 darknessporn.com,familyporner.com,freepublicporn.com,pisshamster.com,punishworld.com,xanimu.com##.no-gutters > .col-6:has(> #special-block)
 kbjfree.com##.relative.w-full:has(> .video-card[target="_blank"])
+lewdzone.com##.row > div:has(> .f8rpo)
```

Comment:
@rakleed Revert this, we don't want to use `:has()` for such a little change.

Reply:
Done cc609ff